AGENTS.md

```diff
@@ -0,0 +1,204 @@
+# OpenShift Monitoring Plugin - AI Agent Guide
+
+## 🚀 Quick Start (30-second overview)
+- **What**: Dual frontend plugins for OpenShift observability (monitoring-plugin + monitoring-console-plugin)
+- **Purpose**: Alerts, Metrics, Targets, Dashboards + Perses, Incidents, ACM integration
+- **Tech Stack**: React + TypeScript + Webpack + i18next
+- **Key Files**: `web/console-extensions.json`, `web/src/components/`
+
+## 📋 Common Tasks & Workflows
+
+### Adding a New Feature
+1. Check if it belongs in `monitoring-plugin` (core) or `monitoring-console-plugin` (extended)
+2. Update console extensions in `web/console-extensions.json`
+3. Add React components in `web/src/components/`
+4. Add translations in `public/locales/`
+5. Test with `make lint-frontend && make test-backend`
+
+### Debugging Issues
+- **Build failures**: Check `Makefile` targets
+- **Console integration**: Verify `console-extensions.json`
+- **Plugin loading**: Check OpenShift Console logs
+- **Perses dashboards**: Debug at `web/src/components/dashboards/perses/`
+
+### Development Setup
+```bash
+# See README.md for full setup
+make build-frontend
+make test-backend
+# Deployment: https://github.com/observability-ui/development-tools/
+```
+
+## 🏗️ Architecture Map
+
+```
+OpenShift Console Platform
+├── monitoring-plugin (CMO managed)
+│   ├── Alerts, Metrics, Targets
+│   └── Legacy Dashboards
+└── monitoring-console-plugin (COO managed)
+    ├── Perses Dashboards
+    ├── Incidents/Health Analyzer
+    └── ACM Multi-cluster
+```
+
+## 🔍 Key Components Reference
+
+| Component | Purpose | Entry Point | Manager |
+|-----------|---------|-------------|---------|
+| monitoring-plugin | Core observability UI | `web/console-extensions.json` | CMO |
+| monitoring-console-plugin | Extended features | COO UIPlugin CR | COO |
+| Perses | Dashboard visualization | `web/src/components/dashboards/perses/dashboard-page.tsx` | Embedded |
+| ACM Integration | Multi-cluster alerts | COO UIPlugin configuration | ACM Operator |
```

Comment:
ACM operator does not manage the plugin, it uses COO to do it.

Reply:
After reviewing this, I removed this section because it felt redundant.